packages/react-native/android/src/main/java/com/dashx/sdk/DashXModule.kt

```diff
@@ -1,8 +1,6 @@
 package com.dashx.sdk
 
-import com.facebook.react.bridge.ReactApplicationContext
-import com.facebook.react.bridge.ReactContextBaseJavaModule
-import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.*
 
 class DashXModule(private val reactContext: ReactApplicationContext) : ReactContextBaseJavaModule(reactContext) {
     private var dashXClient: DashXClient? = null
```

Comment:
Try removing `?`

Reply:
Removing `?` but adding `!!` to force `NullPointerException` if `DashXClient.instance` is `null` is